test/Makefile

```diff
@@ -28,19 +28,24 @@ TRIMMING_ARGS := "JULIA=$(JULIA_EXECUTABLE)" "BIN=$(SRCDIR)/trimming" "CC=$(CC)"
 TEST_JULIA_OPTIONS := --check-bounds=yes --startup-file=no --depwarn=error
 TEST_SCRIPT_OPTIONS := --buildroot=$(call cygpath_w,$(BUILDROOT))
 
+default
```

Comment:
.PHONY?

Reply:
Indeed, thanks, fixed. not sure what happened there.